.pre-commit-config.yaml

```diff
@@ -3,7 +3,7 @@ repos:
     rev: 19.10b0
     hooks:
       - id: black
-  - repo: git://github.com/pre-commit/pre-commit-hooks
+  - repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
It suddenly [started failing](https://github.com/getsentry/snuba/runs/4080712275?check_suite_focus=true):

```
pre-commit install --install-hooks
pre-commit installed at .git/hooks/pre-commit
[INFO] Initializing environment for https://github.com/psf/black.
[INFO] Initializing environment for git://github.com/pre-commit/pre-commit-hooks.
An unexpected error has occurred: CalledProcessError: command: ('/usr/bin/git', 'fetch', 'origin', '--tags')
return code: 128
expected return code: 0
stdout: (none)
stderr:
    fatal: remote error: 
      The unauthenticated git protocol on port 9418 is no longer supported.
    Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.
    
Check the log at /home/runner/.cache/pre-commit/pre-commit.log
make: *** [Makefile:11: setup-git] Error 3
```

Reply:
You can revert this I think, i have a separate PR that upgrades the pre-commit version and that fixes this. https://github.com/getsentry/snuba/pull/2183